### PR TITLE
feat(chip): add read-only variation

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly-next/issues/1553

### CODE REVIEW

**Comment by srambach** [comment]
LGTM 👍

File: src/patternfly/components/Chip/chip.scss [hunk 1]
```diff
@@ -24,6 +24,11 @@
   --pf-c-chip--m-overflow--c-button--focus--BorderWidth: var(--pf-global--BorderWidth--sm);
   --pf-c-chip--m-overflow--c-button--focus--BorderColor: var(--pf-global--secondary-color--100);
 
+  // read-only chip
+  --pf-c-chip--m-read-only--PaddingTop: var(--pf-global--spacer--form-element);
```
Comment (christiemolloy) [inline src/patternfly/components/Chip/chip.scss]: It seems weird to me that the modified version is the simplest chip we have with all padding present..would it be wrong to have read only be the simplified chip component and then have a modifier for the chip with the "remove" button. I guess though the chip is always meant to come with the remove button so this should be fine... im just also thinking if we have chips with icons etc.... you can really put anything into a chip.

File: src/patternfly/components/Chip/examples/chip-example.hbs [hunk 2]
```diff
@@ -23,14 +23,21 @@
...
+    7
+  {{/badge}}
   {{#> button button--modifier="pf-m-plain" button--attribute='aria-labelledby="remove_chip_three chip_three" aria-label="Remove" id="remove_chip_three"'}}
     {{> button-icon button-icon--type="times-circle"}}
   {{/button}}
 {{/chip}}
 <br>
 <br>
+{{#> chip chip--type="div" chip--modifier="pf-m-read-only"}}
+  {{#> chip-text}}
```
Comment (christiemolloy) [inline src/patternfly/components/Chip/examples/chip-example.hbs]: should this chip-text have an id

File: src/patternfly/components/Chip/examples/chip-example.hbs [hunk 3]
```diff
@@ -23,14 +23,21 @@
...
+    Read-only Chip
+  {{/chip-text}}
+{{/chip}}
+<br>
+<br>
 {{#> chip chip--type="div" chip--modifier="pf-m-overflow"}}
   {{#> button button--modifier="pf-m-plain"}}
```
Comment (christiemolloy) [inline src/patternfly/components/Chip/examples/chip-example.hbs]: should this chip-text have an id?

File: src/patternfly/components/Chip/chip.scss (hunk 1 above)
Comment (mcoker) [reply src/patternfly/components/Chip/chip.scss]: >  I guess though the chip is always meant to come with the remove button so this should be fine

That's what I'm thinking, too. Plus that would be a breaking change. Can you give an example of where you would want to use an icon in a chip and how that would affect the layout?

File: src/patternfly/components/Chip/examples/chip-example.hbs (hunk 2 above)
Comment (mcoker) [reply src/patternfly/components/Chip/examples/chip-example.hbs]: I don't think so. The `id` is only used with `aria-labelledby` when there is an icon button (the close button)

File: src/patternfly/components/Chip/examples/chip-example.hbs (hunk 3 above)
Comment (mcoker) [reply src/patternfly/components/Chip/examples/chip-example.hbs]: same as above

File: src/patternfly/components/Chip/chip.scss (hunk 1 above)
Comment (christiemolloy) [reply src/patternfly/components/Chip/chip.scss]: Ok lets stick with this then and avoid a breaking change. I have only seen icons used at the beginning and last element of the chip, but we dont have a case for Patternfly yet so all good

**Pull request merged**